contracts/interfaces/IDiamondCut.sol

```diff
@@ -33,6 +33,4 @@ interface IDiamondCut {
         bytes calldata _calldata
     )
         external;
-
-    event DiamondCut(FacetCut[] _diamondCut, address _init, bytes _calldata);
```

Comment:
This event isn't needed?

Reply:
@cinnabarhorse
Yes, this function is not needed for this interface. i checked other diamonds.